build.sbt

```diff
@@ -77,6 +77,7 @@ lazy val `support-frontend` = (project in file("support-frontend"))
     buildInfoPackage := "app",
     buildInfoOptions += BuildInfoOption.ToMap,
     scalastyleFailOnError := true,
+    fork in run := true,
```

Comment:
If it's likely to have unexpected side effects (probably not) or clutter the main pr too much, it can be worth putting in a separate PR.
If its in this PR it would be good to write a brief comment here (in github is fine) explaining why it's useful/justification and any potential downsides.  That helps reviewers know that it was intentional